The Contact Form Widget (#2376)
umbraco/UmbracoDocs: Add-ons/Starter-Kit-TEMP/Widgets/index.md

- Text Color
- Form (this is where you add the form that you have already created)
- Container Width

### The Contact Form Widget

![movePage.jpg](images/Contact-Form.png)

> The Contact Form Widget will let you add a contact form to your project, however, to make it work you will need to set up a contact email.

Reviewer: ```suggestion
The Contact Form Widget will let you add a basic contact form to your project. To make it work you will need to set up a contact email.
```

> You set up the contact email by going to the settings tab, then you proceed to the general tab, from here you fill out the ***Contact Form Email***.

Reviewer: ```suggestion
You set up the contact email by going to the General Settings node, and fill out the ***Contact Form Email*** field.
```

> ![movePage.jpg](images/settings-contact-form.png)

Reviewer: Instead of having this image here - perhaps we could link to the article describing this setting instead? 🤔 What do you think?
